scripts/normalize_text.py

```
@@ -0,0 +1,97 @@
+        """Normalize a single file.
+
+        Returns:
+            True if file was modified, False otherwise.
+        """
+        if not path.is_file():
+            return False
+        try:
+            # For Python >=3.13: original = path.read_text(encoding="utf-8", newline="")
+            with path.open("r", encoding="utf-8", newline="") as f:
+                original = f.read()
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)